components/ReviewTransactionModal.tsx

```diff
@@ -37,7 +37,7 @@ const ReviewTransactionModal = ({
   tokenSymbol,
   tokenName,
 }: ReviewTransactionModalProps) => {
-  const { txNotify } = useNotifications();
+  const { txNotify, notifyUser } = useNotifications();
```

Comment:
_(arbitrary comment location to start a thread)_

Getting this error in the console, that token address is [MATIC](https://polygonscan.com/address/0x0000000000000000000000000000000000001010), i.e. the native token (which exists at that address since ETH is not the native token IIRC)

<img width="1281" alt="image" src="https://user-images.githubusercontent.com/17163988/214869482-ce64f8e8-2d4f-402d-96c3-159074d972e4.png">

Reply:
Fixed this by removing native token from the token list